pkg/webhook/server/certificate/configuration.go

```diff
@@ -109,5 +109,10 @@ func (m *Manager) CABundle() ([]byte, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to get %s webhook configuration %s", m.webhookType, m.webhookName)
 	}
-	return m.clientConfigList(webhook)[0].CABundle, nil
+
+	if clientConfigList := m.clientConfigList(webhook); len(clientConfigList) == 0 {
+		return nil, errors.Wrapf(err, "failed to get clientConfig in %s webhook configuration %s", m.webhookType, m.webhookName)
+	} else {
+		return clientConfigList[0].CABundle, nil
```

Comment:
Shouldn't we also check if CABundel is not nil?

Reply:
Agree let's add that check to first == nil then len 0 with different error message so we can differenciate.